### PR TITLE
The Kubernetes Dashboard Evolution blog post

### PR DESCRIPTION
**Pull request by floreks**
This blog post is a summary of the Kubernetes Dashboard history along with some future roadmap plans.

cc @maciaszczykm

### CODE REVIEW

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (sftim) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:37]: Hi

I'd prefer to see images defined in Markdown rather than done using rendered HTML.

We can help you make use of Markdown and also Hugo shortcodes so that the article doesn't define HTML directly.

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (sftim) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:80]: With the [CNCF website guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md) in mind, I strongly recommend revising or removing this paragraph and hyperlink.

> It is OK to say that, e.g., “Prometheus was originally created by Soundcloud” or “Kubernetes builds upon 15 years of experience of running production workloads at Google,” but the origin company should not otherwise be referred to on the project homepage.

That's talking about home pages, but I'd like to see the same intent captured in how we publish articles on the blog. 
So, for example, “The Kubernetes Dashboard was originally created by Google” would be fine, but this link to Kubermatic feels like it wouldn't be.


See the [Kubernetes content guide](https://kubernetes.io/docs/contribute/style/content-guide/) for more on this.

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (sftim) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:51]: I would write:
```suggestion
On top of that, we put in a great deal of effort into [improving resource support](https://github.com/kubernetes/dashboard/issues/5232). Meanwhile, we do offer support for most of the Kubernetes resources. Also, the Kubernetes Dashboard supports multiple languages: English, German, French, Japanese, Korean, Chinese (Traditional, Simplified, Traditional Hong Kong). Persian and Russian localizations are currently in progress. Moreover, we are working on the support for 3rd party themes and the design of the app in general. As you can see, quite a lot of things are going on.
```

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (sftim) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:8]: Most commonly we attribute like this:
```suggestion
Authors: Marcin Maciaszczyk, Kubermatic & Sebastian Florek, Kubermatic
```

**Review by sftim — COMMENTED** [review]
Hi @floreks 

Thanks for the article! I've a couple of reservations about accepting this; the things I think we'll need you to address before this can merge are:
- the final paragraph with the link to Kubermatic
- the inline HTML (we can help you to use Markdown instead)

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (floreks) [reply content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:80]: Sure, that's totally understandable. We weren't sure if this would be accepted anyway. I will simply remove that part.

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (floreks) [reply content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:37]: I have tried to use original markdown/hugo, but with the current theme it wouldn't really allow to center images + use nice caption for them. I am open to suggestions here.

Another option was to add a custom shortcode with custom styling that would allow to better handle image positioning and caption, but it would require more extensive changes outside of only the blog directory, so I was hesitant to do so.

Comment (sftim) [reply content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:37]: Is `imgproc` from Docsy good enough?

https://www.docsy.dev/docs/adding-content/shortcodes/#imgproc

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (sftim) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:78]: _optional tweak_
```suggestion
As mentioned earlier, we are currently looking for more people to help us further develop and grow the project. We are open to contributions in multiple areas, i.e., [issues with help wanted label](https://github.com/kubernetes/dashboard/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22). Please feel free to reach out via GitHub or the #sig-ui channel in the [Kubernetes Slack](https://slack.k8s.io/).
```
?

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (floreks) [reply content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md]: It's ok'ish I guess. It is quite limited and requires manually tweaking the size (width, height). Also the position argument `left|center|right` does not seem to work. I have tried to make it look as best as possible given the limited functionality.

**Comment by sftim** [comment]
@kubernetes/sig-docs-blog-reviewers PTAL

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (floreks) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:40]: I have updated latest release info to `v2.2.0` as it was released recently.

Comment (maciaszczykm) [reply content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:40]: How does it work with the date of the post?

Comment (floreks) [reply content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:40]: Date of the release 18.02
Date of the blog post 22.02

Comment (sftim) [reply content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:40]: Once this is ready for review we can push a commit to match the date it'll go live.

We can accept future-dated articles but past dated ones I think we prefer not to (there's no CI test or anything, it's just odd for readers when an article turns up today that says it was published last week).

**Comment by floreks** [comment]
Any updates?

**Comment by maciaszczykm** [comment]
Hi, are there any updates?

**Comment by sftim** [comment]
(Preview](https://deploy-preview-26668--kubernetes-io-master-staging.netlify.app/blog/2021/02/22/the-evolution-of-kubernetes-dashboard/) looks OK to me.

**Comment by sftim** [comment]
LGTM. We'll need to adjust the date before publication.

**Comment by floreks** [comment]
@sftim what would be the next step then? Wait for someone to ping me about the date update?

**Comment by maciaszczykm** [comment]
@floreks I think you should do it.

**Comment by floreks** [comment]
I have updated the date.

**Comment by mrbobbytables** [comment]
@floreks @maciaszczykm I'll review shortly, we should be able to get this out today 👍

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (mrbobbytables) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:10]:
```suggestion
In October 2020, the Kubernetes Dashboard officially turned five. As main project maintainers, we can barely believe that so much time has passed since our very first commits to the project. However, looking back with a bit of nostalgia, we realize that quite a lot has happened since then. Now it’s due time to celebrate “our baby” with a short recap.
```
minor non-blocking nits :) Just think it helps improve the readability.

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (mrbobbytables) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:14]:
```suggestion
The initial idea behind the Kubernetes Dashboard project was to provide a web interface for Kubernetes. We wanted to reflect the kubectl functionality through an intuitive web UI. The main benefit from using the UI is to be able to quickly see things that do not work as expected (monitoring and troubleshooting). Also, the Kubernetes Dashboard is a great starting point for users that are new to the Kubernetes ecosystem.
```

Few other minor nits for readability.

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (mrbobbytables) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:36]:
```suggestion
The migration came with many benefits like being able to refactor a lot of the code, introduce design patterns, reduce code complexity, and benefit from the new modules. However, you can imagine that the scale of the migration was huge. Luckily, there were a number of contributions from the community helping us with the resource support, new Kubernetes version support, i18n, and much more. After many long days and nights, we finally released the [first beta version](https://github.com/kubernetes/dashboard/releases/tag/v2.0.0-beta1) in July 2019, followed by the [2.0 release](https://github.com/kubernetes/dashboard/releases/tag/v2.0.0) in April 2020 — our baby had grown up.
```
non-blocking nit

File: content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md
Comment (mrbobbytables) [inline content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md:48]:
```suggestion
The Kubernetes Dashboard has been growing and prospering for more than 5 years now. It provides the community with an intuitive Web UI, thereby decreasing the complexity of Kubernetes and increasing its accessibility to new community members. We are proud of what the project has achieved so far, but this is by far not the end. These are our priorities for the future:
```

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks!
I have a few non-blocking suggestions - but one thing that will be needed before merge is updating the date in the name the directory to match the published date.

Once that's done (and if you want to address the nits) I'm good to approve publication 👍

**Comment by maciaszczykm** [comment]
@sftim Can we get `/lgtm`?

**Comment by mrbobbytables** [comment]
I'll apply lgtm, as he did it in comment earlier pre-date change.

/lgtm

**Comment by floreks** [comment]
FYI @mrbobbytables @sftim 
Preview: https://deploy-preview-26668--kubernetes-io-master-staging.netlify.app/blog/2021/03/09/the-evolution-of-kubernetes-dashboard/
vs
Live: https://kubernetes.io/blog/2021/03/09/the-evolution-of-kubernetes-dashboard/

Not sure why but images are not rendered in the live version.

**Comment by kbhawkey** [comment]
Hi @floreks .
Is it possible to create new images, scaled to the appropriate size or link to a site that displays the images?

**Comment by maciaszczykm** [comment]
What is the appropriate size?

**Comment by sftim** [comment]
> appropriate size

Make sure that the size that the image shows at in the preview matches the original image size. This _should_ mean that imgproc doesn't have to scale.

Instead of that, maybe we can suggest a different shortcode?

_Is there an issue logged to track these missing images and the work to reinstate them?_

**Comment by floreks** [comment]
@sftim it shouldn't be hard to manually rescale the images and commit them instead of existing ones. We will take care of that and prepare a PR.

**Pull request merged**
